asana/asana.go

```diff
@@ -272,7 +283,7 @@ func (c *Client) request(method string, path string, data interface{}, opt *Filt
 	res := &Response{Data: v}
 	err = json.NewDecoder(resp.Body).Decode(res)
 	if len(res.Errors) > 0 {
-		return res.Errors[0]
+		return res.Errors
```

Comment:
According to this, at least 1 element. I think it's worth documenting that for the `Errors` type, so users will know to always expect either at least one error there.